src/components/calculator.js

```diff
@@ -11,6 +11,9 @@ const CalcInterface = () => {
   const { total, next, operation } = state;
   return (
     <section>
+      <div className="practice">
+        <p>let&apos;s do some maths</p>
+      </div>
       <div className="calc-container">
         <div className="header">
           <button type="button" className="p-tag">
```

Comment:
- [ ] [Optional] your design is great but if possible I would suggest you please use the given wireframes. See the image below for reference
### Original design
![your](https://user-images.githubusercontent.com/81727988/206156838-c486261c-e0ba-4830-8deb-cdb78dd9b56f.png)
### Your design
![orignal](https://user-images.githubusercontent.com/81727988/206156891-7a374a45-7906-482a-a9d9-f1050b35c3cb.png)

Reply:
Hi @roshan-bajgain good to hear from you, Thank you for your feedback I really appreciate it. I shall implement the requested modifications to my project.